packages/react-internal/src/components/Stack/StackItem/StackItem.types.ts

```diff
@@ -42,7 +42,8 @@ export type IStackItemStylesReturnType = ReturnType<Extract<IStackItemComponent[
  */
 export interface IStackItemProps
   extends IStackItemSlots,
-    IStyleableComponentProps<IStackItemProps, IStackItemTokens, IStackItemStyles> {
+    IStyleableComponentProps<IStackItemProps, IStackItemTokens, IStackItemStyles>,
+    React.HTMLAttributes<HTMLElement> {
```

Comment:
you are referencing `React` without explicitly importing it from top. could you please add import statement for `React` and re-generate the API file

Reply:
Just added the import statement in the latest push